classes/components/forms/emailTemplate/EmailTemplateForm.php

```diff
@@ -46,6 +48,14 @@ public function __construct(string $action, array $locales)
                 'isMultilingual' => true,
                 'toolbar' => 'bold italic superscript subscript | link | blockquote bullist numlist',
                 'plugins' => 'paste,link,lists',
+            ]))->addField(new FieldEmailTemplateUnrestricted('isUnrestricted', [
```

Comment:
I think a lot more of the setup logic for the Vue components can be done in PHP so that these can both be done with existing `FieldOptions` components. I mentioned this in the `ui-library` PR as well and we can chat about it more in our next one-on-one (probably easier to share screen and code together than writing it all out here).